src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -660,6 +690,23 @@ def _get_clone_source(self):
             raise VolumeException(-errno.EINVAL, "error fetching subvolume metadata")
         return clone_source
 
+    def _get_clone_progress(self):
+        try:
+            src_snappath = self.metadata_mgr.get_option("source", "snappath")
+            src_usedbytes = int(self.fs.getxattr(src_snappath, 'ceph.dir.rbytes'))
```

Comment:
I guess you could also use, self.metadata_mgr.get_option("source", "size")?

Reply:
Done